filters/regional/filter_18_ITA_FiltriDNS/configuration.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ITA: Filtri-DNS",
-  "description": "List for blocking ads on Italian websites based on Easylist Italy and ADB X Files",
+  "description": "List for blocking ads on Italian websites based on EasyList Italy and ADB X Files",
```

Comment:
https://github.com/easylist/easylist
they use both Easylist and EasyList

Reply:
@Alex-302 EasyList Italy used at the Crowdin. But EasyList is better I guess.

![image](https://user-images.githubusercontent.com/35523759/212072292-5075ab7b-7847-4ade-88d3-41a88f863ce2.png)